### DISCUSSION TITLE
Could you supply method to view error log in the Submit result page?

### DISCUSSION THREAD

**Discussion by zhaizhixiang** [PaddlePaddle/PaddleSpatial#103] [BaiduKDDCup2022] [answered]
Could you supply method to view error log in the Submit result page? That will be a more convenient way for debuging the code issue than that asking your help every time.

**Comment by ramber1836** [comment, marked as answer]
Due to the risk of sensitive information exposure, we cannot let you view the full error log. 
Thanks for your understanding.
